config/environment.js

```diff
@@ -56,6 +54,9 @@ module.exports = function(environment) {
 
   if (environment === 'production') {
     // Some prod-only config
+    ENV.API = {
+      host: 'https://conduit.productionready.io'
```

Comment:
Should we do dev against Mirage? Or the real API? I kinda of figured it should be the API